packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -192,12 +192,11 @@ class MariaDbInstanceEngine extends InstanceEngineBase {
     });
   }
 
-  public bindToInstance(_scope: core.Construct, _options: InstanceEngineBindOptions): InstanceEngineConfig {
-    const config = super.bindToInstance(_scope, _options);
-    if (_options.domain) {
+  public bindToInstance(scope: core.Construct, options: InstanceEngineBindOptions): InstanceEngineConfig {
```

Comment:
Sorry, was trying to model off of the `bindToInstance` method from the `SqlServerInstanceEngineBase` class which has the parameters with underscores. Is that because the method in that class doesn't use them?

Reply:
Yes, exactly 🙂. No worries!